experiments/autoencoders/autoencoders_simple.py

```diff
@@ -13,6 +13,8 @@
 
 
 class DenseNet(nn.Module):
```

Comment:
could we import this one from `.densenet` as well?

Reply:
(and similarly the others)
```
from .configs import get_task_cfg
from .dataloaders import get_dataloaders
```

Reply:
yeah, I suppose we could.  I was originally thinking of making autoencoders_simple a standalone file.

Reply:
gotcha. then we can go ahead and merge